pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -300,7 +300,7 @@ func applyCounterOrListPolicy(c *autoscalingv1.CounterPolicy, l *autoscalingv1.L
 		bufferSize = l.BufferSize
 	}
 
-	// Checks if we've limited by TOTAL capacity
+	// Checks if we've limited by AVAILABLE capacity
```

Comment:
Total for all the ones for fleet autoscaler.
```suggestion
	// Checks if we've limited by total capacity
```